openlibrary/components/MergeUI.vue

```diff
@@ -62,9 +62,14 @@ export default {
         async doMerge() {
             if (!this.$refs.mergeTable.merge) return;
             const { record: master, dupes, editions_to_move } = this.$refs.mergeTable.merge;
+            const unretrieved_editions_works = await this.$refs.mergeTable.find_unretrieved_editions;
 
             this.mergeStatus = 'Saving...';
             try {
+                if (unretrieved_editions_works.length)
+                {
+                    throw new Error(`Could not merge: ${unretrieved_editions_works.join(', ')} has more than ${default_edition_limit} editions.`);
```

Comment:
We also want to alert here so the user gets the error.
```suggestion
                    alert(`Could not merge: ${unretrieved_editions_works.join(', ')} has more than ${default_edition_limit} editions.`);
                    throw new Error(`Could not merge: ${unretrieved_editions_works.join(', ')} has more than ${default_edition_limit} editions.`);
```